### DISCUSSION TITLE
Main

### DISCUSSION THREAD

**Discussion by Toed115** [Seven0492/Resume#5] [General]
there was an embarrassing amount of typos in there

**Comment by Seven0492** [comment]
Lol, indeed :P

**Comment by Seven0492** [comment]
I'm contemplating removing temporarily the reviewer write access limit until you get better and read my messages. I've sent you a proposal for better and more extensive permission for this to make this work. It should be in your Github notifications.

**Comment by Seven0492** [comment]
I'm going to remove it. Tell me clearly when you feel better, enough for this.
